packages/utils/src/lib/throttle.ts

```diff
@@ -68,6 +78,8 @@ export function fpsThrottle(fn: () => void) {
 		}
 		tick()
 	}
+	throttledFn.cancel = () => frame && cancelAnimationFrame(frame)
```

Comment:
Not sure we can cancel here. This would cancel all the functions in the `fpsQueue`, which we probably don't want?

What we can do though is filter out the function from the queue?